### ISSUE TITLE
Re-click the All Ads button

### ISSUE THREAD

**Issue by Pantyhose-X** [dhowe/AdNauseam#2410]
### Describe the issue
In the view ad vault inside, add a Re-Click All Ads button

**Comment by mneunomne** [comment]
Good idea. Maybe then it would make sense to add additional features to the vault such as:

- filter by domain
- filter by ad-network
- filter by clicked / not-clicked

**Comment by ice-fly** [comment]
I actually fundamentally disagree with this feature and believe it is against the original whitepaper/adnauseam spirit: By allowing users to revisit all adds or manually click ads it lowers the obfuscation of the user and makes it easier to identify this ad-fuscation method. Rather by only 'naturally clicking' with some probability like the adnauseam UI currently allows it creates a harder to characterize trail of ad blocking and advert profile disruption.
